### PR TITLE
Alex/image classification

### PR DESCRIPTION
**Pull request by alex936**

### CODE REVIEW

File: image-classification/content.md [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
-```
 
+## Make an isolated Python environment
+Run ```conda create --name image-classification python=3.6``` in your terminal.
+Then, to activate your new environment, run ```conda activate image-classification```.
+
+
+##  Install the required packages
+
+Run ```pip install mlflow[extras] mflux-ai Keras=2.2```  in your terminal.
```
Comment (iver56) [inline image-classification/content.md]: `Keras==2.2`, not `Keras=2.2`

File: image-classification/content.md [hunk 2]
```diff
@@ -27,10 +39,12 @@ from keras.layers import Dense, Dropout, Flatten
 from keras.layers import Convolution2D, MaxPooling2D
 from keras.datasets import mnist
 from keras import optimizers
+
+import mflux-ai
```
Comment (iver56) [inline image-classification/content.md]: `mflux_ai`, not `mflux-ai`

**Review by iver56 — CHANGES_REQUESTED** [review]

**Review by iver56 — APPROVED** [review]
